dask_cuda/utils.py

```diff
@@ -972,3 +988,41 @@ def convert(self, value, param, ctx):
                 choices_str = ", ".join(f"'{c}'" for c in self.choices)
                 self.fail(f"invalid choice(s): {v}. (choices are: {choices_str})")
         return values
+
+
+def _get_active_ucx_implementation_name(protocol):
+    """Get the name of active UCX implementation.
+
+    Determine what UCX implementation is being activated based on a series of
+    conditions. UCXX is selected if:
+    - The protocol is `"ucxx"`, or the protocol is `"ucx"` and the `distributed-ucxx`
+      package is installed.
+    UCX-Py is selected if:
+    - The protocol is `"ucx-old"`, or the protocol is `"ucx"` and the `distributed-ucxx`
+      package is not installed, in which case a `FutureWarning` is also raised.
```

Comment:
This `FutureWarning` is happening elsewhere? Or has that not been implemented yet?

Reply:
[The `FutureWarning` is the one that happens in rapids-dask-dependency](https://github.com/rapidsai/rapids-dask-dependency/blob/88d91b634422c4c87e1db6fc85a3410c7685a351/rapids_dask_dependency/patches/distributed/comm/__rdd_patch_ucx.py#L566-L580).